policies/trademark-policy.md

```diff
@@ -1,172 +1,335 @@
+- Unacceptable: Morphia **by** *&lt;your company name>*
+- Unacceptable: OpenRewrite **-** *&lt;your product name>*
+- Unacceptable: *&lt;your product/company name>* **-** Quarkus
 
-- Do Not Use CF Marks that are not listed, or use Forms of CF Marks that vary from the list. Even a seemingly innocuous deviation may create a new and different trademark.
-    - Do not shorten, abbreviate or create acronyms out of CF Trademarks.
-    - A multiword trademark should never be compressed into one word or vice versa.
-    - Check the list of CF Marks at <https://www.commonhaus.org/trademarks/> to verify exactly how the marks should appear.
+> [!NOTE]
+>
+> For plugins, extensions, and other add-on software, the "likelihood of confusion" concern runs counter to how we speak. Let's use Quarkus extensions as an example.
+>
+> It may seem normal to say "Alice's Quarkus extension", but "Quarkus extension" can be understood two different ways. That phrase could be interpreted to mean either "Alice's extension for Quarkus" or "an extension for Alice's Quarkus". By using this form, you've introduced a possible misunderstanding about the source of Quarkus. The prepositional form, "Alice's extension for Quarkus" avoids that ambiguity.
 
-- Do not use CF Marks (Logos or Word Marks) in any commercial or marketing context other than as expressly permitted in this policy unless you have obtained explicit written permission from the Commonhaus Foundation to do so.
+#### Use of trademarks to show community affiliation
 
-- Do not use the CF Marks in a manner that would disparage the Commonhaus Foundation or its projects (e.g., untruthful advertising, false/misleading promotional materials, etc.).
+This section outlines the use of our Marks in community contexts, such as avatars, decorative skins, or themes. Guidelines for using our Marks on websites are addressed separately below.
 
-- Do not use the Commonhaus Foundation trademarks more prominently than your own company, product or service name.
+You are welcome to use our Word Marks and Logos in non-commercial, decorative contexts—such as avatars, themes, or personas—to show your support for the Project, provided that your usage does not imply official representation, endorsement, or affiliation with the Foundation, Project, or community.
 
-- Do not refer to a product or service as being certified under any CF Mark unless your company has explicit authorization to use such terms from the Commonhaus Foundation.
+#### Websites
 
-- Do not attempt to claim or assert any ownership rights in any mark of the Commonhaus Foundation and do not attempt to register any Commonhaus Foundation trademark as a trademark, trade name, domain name, or “doing business as” name, alone or (unless specifically licensed) in combination with your own trademarks.
+You may use the Word Marks and Logos, but not the Trade Dress, on your webpage to show your support for the Foundation or its Projects as long as:
```

Comment:
Can we link to a definition of "Trade Dress" here, as it's not a term I'd heard of previously

Reply:
```suggestion
You may use the Word Marks and Logos, but not the [Trade Dress](https://www.inta.org/topics/trade-dress/), on your webpage to show your support for the Foundation or its Projects as long as:
```